### PR TITLE
Homework1

### PR DESCRIPTION
**Pull request by dykyys**
completed homework 1 for node

### CODE REVIEW

File: contacts.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const contacts = await listContacts();
+  const contactById = contacts.find((contact) => contact.id === contactId);
+  return contactById;
+}
+
+async function removeContact(contactId) {
+  const contacts = await listContacts();
+  const result = contacts.filter((contact) => contact.id !== contactId);
+  const resultAsJSON = JSON.stringify(result);
+  fs.writeFile(contactsPath, resultAsJSON);
```
Comment (odykyi) [inline contacts.js]: await

File: contacts.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const id = contacts.length ? [...contacts].pop().id + 1 : 1;
+  const newContact = {
+    id,
+    name,
+    email,
+    phone,
+  };
+  contacts.push(newContact);
+  const contactAsJSON = JSON.stringify(contacts);
+  fs.writeFile(contactsPath, contactAsJSON);
```
Comment (odykyi) [inline contacts.js]: await

**Review by odykyi — CHANGES_REQUESTED** [review]
comments

**Comment by dykyys** [comment]
I added await

**Review by odykyi — APPROVED** [review]

**Pull request merged**